cmd/network-observer/profiles.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"github.com/skupperproject/skupper/cmd/network-observer/internal/flowlog"
+	"github.com/skupperproject/skupper/pkg/vanflow"
+)
+
+var (
+	// loggingProfileMinimal logs 1 vanflow event per second (with bursts up to 32)
+	// reduces Link Record noise to 1 every ~20s.
+	// excludes network flow records
+	loggingProfileMinimal = []flowlog.Rule{
+		{
+			Priority: 5,
+			Match: flowlog.NewRecordTypeSet(
+				vanflow.SiteRecord{}, vanflow.RouterRecord{},
+				vanflow.ProcessRecord{}, vanflow.ConnectorRecord{},
+				vanflow.LinkRecord{}, vanflow.ListenerRecord{},
```

Comment:
Should LinkRecord be part of both rules (priority 5 and 1)?

Reply:
Ah, there was no real sense in that. Dropped it from the p 5 rule (lowest wins). Thank you!